src/main/resources/translations/en.json

```diff
@@ -497,6 +497,8 @@
 	"display.suggest.classerrors.desc": "Show suggestion popups when certain errors occur",
 	"display.maxrecent.name": "Max recent files",
 	"display.maxrecent.desc": "How many items to show in recent files menu.",
+	"display.usesystemmenubar.name": "System menubar",
+	"display.usesystemmenubar.desc": "Use system menubar if it is possible.",
```

Comment:
Can add some additional context, perhaps something like `Use system menubar on MacOS`

Reply:
I made a description like this because as I know, some Linux environments have a system menubar too, but I'm not sure.  😖